src/PublicApi/Hooks.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace WPML\PB\PublicApi;
+
+use WPML_PB_Last_Translation_Edit_Mode;
+
+class Hooks {
+
+	/** @var WPML_PB_Last_Translation_Edit_Mode $lastEditMode */
+	private $lastEditMode;
+
+	public function __construct( WPML_PB_Last_Translation_Edit_Mode $lastEditMode ) {
+		$this->lastEditMode = $lastEditMode;
+	}
+
+	public function addHooks() {
+		add_filter( 'wpml_pb_post_use_translation_editor', [ $this, 'postUseTranslationEditor' ], 10, 2 );
+	}
+
+	/**
+	 * @param bool $incomingValue
+	 * @param int  $postId
+	 *
+	 * @return bool
+	 */
+	public function postUseTranslationEditor( $incomingValue, $postId ) {
```

Comment:
@strategio I think we usually called the first parameter something like `$ignored`. Not sure exactly, but I think you can find a few examples in our code base.
”incomingValue” is not clear enough, in my opinion.